cli/tools/installer.rs

```diff
@@ -139,6 +139,57 @@ pub fn infer_name_from_url(url: &Url) -> Option<String> {
   Some(stem)
 }
 
+pub fn uninstall(name: String, root: Option<PathBuf>) -> Result<(), AnyError> {
+  let root = if let Some(root) = root {
+    canonicalize_path(&root)?
+  } else {
+    get_installer_root()?
+  };
+  let installation_dir = root.join("bin");
+
+  // ensure directory exists
+  if let Ok(metadata) = fs::metadata(&installation_dir) {
+    if !metadata.is_dir() {
+      return Err(generic_error("Installation path is not a directory"));
```

Comment:
Another minor point: It might be useful to the user to include the resolved installation_dir path in this error message.